sdk/core/Azure.Core/src/Shared/Autorest/ManagementPipelineBuilder.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text.Json;
+using System.Xml;
+using Azure.Core.Pipeline;
+
+namespace Azure.Core
+{
+    internal static class ManagementPipelineBuilder
+    {
+        public static HttpPipeline Build(TokenCredential credential, ClientOptions options)
+        {
+            return HttpPipelineBuilder.Build(options, new BearerTokenAuthenticationPolicy(credential, "https://management.azure.com//.default"));
```

Comment:
Probably not a big deal since they usually get canonicalized, but did you mean for two "//" here toward the end of the URI?

Reply:
Yeah, actually @schaabs told me to do it :)

Reply:
It's not technically a URI it's a scope string, which has to be exact. Since the resource string registered with AAD, "https://management.azure.com/", ends in a "/" the default scope has the two "//". It's unfortunate because it's inconsistent between services, and is dependent on how the resource was registered with AAD. For instance the resource for key vault is "https://vaults.azure.net" doesn't end in a "/" so its default scope, "https://vaults.azure.net/.default" does not have the double slashes.